app/src/main/java/org/mozilla/focus/home/HomeTilePlaceholderGenerator.kt

```diff
@@ -12,21 +12,23 @@ import android.net.Uri
 import android.support.v4.content.ContextCompat
 import android.util.TypedValue
 import org.mozilla.focus.R
+import org.mozilla.focus.ext.withRoundedCorners
 import org.mozilla.focus.utils.UrlUtils
 
 class HomeTilePlaceholderGenerator {
 
     companion object {
-        private val TEXT_SIZE_DP = 52f
+        private val TEXT_SIZE_DP = 22f
+        private val DEFAULT_TILE_CORNER_RADIUS = 12f
         private val DEFAULT_ICON_CHAR = '?'
 
         fun generate(context: Context, url: String?): Bitmap {
             val startingChar = getRepresentativeCharacter(url)
 
             // When this method is called, the view we're setting this bitmap on has no size (maybe
-            // because it doesn't contain an image?) so we use a hard-coded estimated size instead.
-            val width = context.resources.getDimensionPixelSize(R.dimen.home_tile_icon_estimated_width)
-            val height = context.resources.getDimensionPixelSize(R.dimen.home_tile_icon_estimated_height)
+            // because it doesn't contain an image?) so we use a hard-coded size instead.
+            val width = context.resources.getDimensionPixelSize(R.dimen.home_tile_placeholder_icon_size)
```

Comment:
nit: `val dimen = ...` and use this dimen var twice, rather than accessing the resource off disk twice.